apps/sms/views/conversation/js/conversation.js

```diff
@@ -672,7 +676,8 @@ var ConversationView = {
     }
 
     // TODO move most of back() here: Bug 1010223
-    if (!this.isConversationPanel(Threads.currentId, nextPanel)) {
+    if (this.activeThread &&
+      !this.isConversationPanel(this.activeThread.id, nextPanel)) {
```

Comment:
nit: can you indent this so that `!this` is just below `this` ? (so 2 characters right -- not 1)